src/Hit/Git.hs

```diff
@@ -73,6 +73,10 @@ runNew issueOrName = do
                        || isSpace c
                        || c `elem` ("_-./" :: String)
                    )
+runNew _ title =
+    createIssue' title >>= \case
+        Left err -> errorMessage $ show err
+        Right issue -> putTextLn . showIssueName blueCode $ issue
```

Comment:
Yeah, I wouldn't put it into the separate case as it should share most of the code with the previous clause.

Reply:
@vrom911 Could you please elaborate on this comment? I'm not sure I understand it correctly

Reply:
What I meant is that this should share a lot of code with the `runNew` :slightly_smiling_face:  Sorry for the poor wording..